README.md

```diff
@@ -16,20 +16,22 @@ The [draft site](https://draft-openlibertyio.mybluemix.net/) is built using the
+* Update the `./scripts/build_clone_docs.sh vX.0.0.X` command in the [./scripts/build_jekyll_maven.sh](https://github.com/OpenLiberty/openliberty.io/blob/development/scripts/build_jekyll_maven.sh) file to clone the latest doc version to get the javadocs which are not handled completely by Antora yet. 
+* Update the version of `{replace_version page.url 'X.0.0.X' }` in [src/main/content/antora_ui/src/partials/head-info.hbs](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/antora_ui/src/partials/head-info.hbs) so that the canonical meta tag is set to the latest version of the documentation for all versions of the docs. 
+* Update the version of the url `<loc>https://openliberty.io/docs/X.0.0.X/overview.html</loc>` in the [src/main/content/sitemap.xml](https://github.com/OpenLiberty/openliberty.io/blob/master/src/main/content/sitemap.xml).
 
-7. (Temporary for now) Rebuild the [development site](http://dev-openlibertyio.mybluemix.net/) using the [IBM Cloud development pipeline](https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350?env_id=ibm:yp:us-south) and test that the docs link works and redirects to the newest doc version once the site is done building.
+7. Make a pull request from your branch into the `development` branch. Once the pull request has been reviewed merge it in. The [development site](http://dev-openlibertyio.mybluemix.net/) should automatically rebuild once the pull request has merged in. It will take ~40 minutes for the changes to show up on the development site. You can track the progress using the [IBM Cloud development pipeline](https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350?env_id=ibm:yp:us-south). Test that the docs link works and redirects to the newest doc version once the site is done building.
 
-8. (Temporary for now) Make a pull request from the [UI development branch](https://github.com/OpenLiberty/openliberty.io/tree/development) into the [UI master branch](https://github.com/OpenLiberty/openliberty.io/tree/master).
-
-9.  Rebuild the openliberty.io site using the IBM Cloud pipeline(https://cloud.ibm.com/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south).
+8. Before proceeding, make sure all of the pull requests and changes from previous steps have been merged into their respective repos. Make a pull request from the [UI development branch](https://github.com/OpenLiberty/openliberty.io/tree/development) into the [UI master branch](https://github.com/OpenLiberty/openliberty.io/tree/master). The [openliberty.io site](https://openliberty.io/) will automatically rebuild when the pull request is merged in. You can track the progress using the IBM Cloud pipeline(https://cloud.ibm.com/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south).
```

Comment:
Should the part about checking all previous prs are merged in go in step 7 so that everything is good for the development build?